packages/eslint-config-nextjs-ts/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    extends: ['plugin:@next/next/recommended', 'prettier', '@infinumjs/eslint-config-react-ts'],
```

Comment:
I am still not sure of the order here. I tried all the combinations, and only this one does not override our rules from our `eslint-config-react-ts` package.

For prettier, I am also not sure should we include it or not. NextJS docs recommend it, but it adds additional rules for e.g. Vue.js (I didn't find a way how to exclude some namespaces from external package).
They recommend to put prettier last (https://github.com/prettier/eslint-plugin-prettier#recommended-configuration), but same as for the NextJS rules, our gets overridden.

And before it was not working since we were using incorrect plugin: https://nextjs.org/docs/basic-features/eslint#recommended-plugin-ruleset
`plugin:@next/next/recommended` should be used, and not `next/core-web-vitals`. `next/core-web-vitals` is only if you DO NOT have your own rules.

I tried this in React Example and with new clone of NextJS (12.1.5) and it works.

You can also see and make a diff for both configs (go to the PR original comment)

Reply:
I would also omit the Prettier config. I would include in projects that use Prettier.

Reply:
Done
https://github.com/infinum/js-linters/pull/66/commits/fdae3353fa07f3ea63f55ec2715880bde1c3ef90